ground/gcs/src/plugins/uavobjectbrowser/treeitem.cpp

```diff
@@ -45,7 +45,7 @@ HighLightManager::HighLightManager(long checkingInterval, QTime *currentTime)
     m_expirationTimer.start(checkingInterval);
     connect(&m_expirationTimer, &QTimer::timeout, this, &HighLightManager::checkItemsExpired);
 
-    if (currentTime == NULL)
+    if (m_currentTime == NULL)
         m_currentTime = new QTime;
```

Comment:
So I've read the surrounding code.  The idea is, one class has an idea of the current time.. that it updates with a timer.. that everyone else has pointers to cuz LOL.